src/sass/main.scss

```diff
@@ -87,3 +87,8 @@ $code-kbd-font-weight: false;
 .show {
   display: block !important;
 }
+
+.nobull {
+  list-style-type: none;
+ }
```

Comment:
Is this used anywhere?

Reply:
It was used in the ErrorTooltip component, but that has since been removed

Reply:
Let's remove the dead code.